app/src/components/Selection.js

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react";
+
+function Selection({ handleAreaSearch }) {
+  const [selectedVal, setSelectedVal] = useState("");
+  const [area, setArea] = useState([
+    {
+      name: "Alabama",
+      abbreviation: "AL",
+    },
```

Comment:
This looks like the kind of data you could even put into its own file, maybe even call it a `json` file, since it's just data and no algorithm.